ietf-te-mpls.yang

```diff
@@ -425,6 +453,13 @@ module ietf-te-mpls {
     uses tunnel-bandwidth_top;
   }
 
+  /* MPLS TE path augmentations */
+  augment "/te:te/te:tunnels/te:tunnel/" +
+          "te:primary-paths/te:primary-path" {
```

Comment:
**2020-07-03 TEAS call**

Move this augmentation under the tunnel

Reply:
Resolved in https://github.com/tsaad-dev/te/pull/104/commits/12dee6df3b3aa1f5aa70d8dd5d51e252c13360b8